_extras/learner_profiles.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Learner Profiles"
+---
+
```

Comment:
Suggest adding something like this (I believe from the Caprentries lesson dev manual - or link to the lesson dev manual?

> These profiles describe the potential learners that we anticipate as learners  
for this lesson. These can be used if you are deciding if this material is  
right for you or your students. If you plan to contribute material to this  
lesson, these will help you understand the target audience so that we can have  
a collaboratively developed, but cohesive lesson.